deployment/windows_menu.json

```diff
@@ -5,7 +5,7 @@
             {
                 "script": "${PREFIX}/Scripts/napari.exe",
 				"name": "micro_sam",
-				"icon": "${PREFIX}/Menu/micro-sam-logo.ico",
+				"icon": "${PREFIX}/Menu/logo.png",

```

Comment:
I think you have to convert this to the .ico file format. Otherwise this will not be displayed correctly.
See https://en.wikipedia.org/wiki/ICO_(file_format) .

Reply:
i used imagemagick cli tool to convert png to ico with: `convert logo.png -define icon:auto-resize=64,48,32,16 logo.ico`